src/librustc/sir.rs

```diff
@@ -135,10 +135,10 @@ impl SirFuncCx {
         }
     }
 
-    fn lower_local(&self, place: &mir::Place<'_>) -> ykpack::Local {
+    fn lower_local(&self, local: &mir::Local) -> ykpack::Local {
```

Comment:
Sorry, one more thing. There's no point in passing the local as a reference. It's small and implements `Copy`, so let's pass it by value.

Reply:
Doesn't `Local(x)` already make a copy of `x`?

Reply:
Well yes, it does. But note that passing a reference copies a pointer address under the hood too!

I'm just trying to reduce syntax noise really. You can use a reference if you feel strongly.

Reply:
Fair.